script.js

```diff
@@ -0,0 +1,18 @@
+// -----------------Storing information in Js object --------------------------------
+const project = {
+  name: 'portfolio',
+  description:
+    'Personal portfolio that contains project information, skills, and contact',
+  technologies: ['HTML', 'CSS', 'JavaScript', 'Bootstrap'],
+  liveVersion: 'https://ridabensalem.github.io/',
+  sourceCode: 'https://github.com/ridabensalem/ridabensalem.github.io',
+};
```

Comment:
- [ OPTIONAL ] From the project requirement, You need to use a JavaScript array to store all of the information for all projects. So I think you should have an array of objects where each object stores the information of each project. 
- Each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Ie:
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
// another project info
},
{
// another project info
},
]
```